### DISCUSSION TITLE
Mermaid objects cutting off text in README.md in Safari

### DISCUSSION THREAD

**Discussion by stoutgz** [community/community#12523] [Other Feature Feedback, Questions, & Ideas]
Using mermaid in markdown in my READMEs and seems suddenly this week it no longer renders correctly specifically in Safari and also specifically in the repo README.md file. I can't reproduce in other browsers, nor @ gist.github.com.

This is how I reproduce:

```
graph TB
    top(Top project) --> sub1("Sub<br>Repo 1<br>⭐️ You are here!")
    top --> sub2(Sub<br>Repo 2)
    top --> sub3(Sub Repo 3)
    sub2 --> sub4(Sub Repo 4)
    sub2 --> sub5(Sub Repo 5)
    top --> sub6(Sub Repo 6)
    top --> sub7(Sub Repo 7)
    top --> sub8(Sub Repo 8)
    
    classDef here fill:#0000FF,stroke:#333,stroke-width:2px;
    class sub2 here
```

macOS 12.2 (M1 Pro)

- Safari 15.3
    + In repo README.md, looks like this:
        * ![Screen Shot 2022-03-07 at 3 59 06 PM](https://user-images.githubusercontent.com/3385153/157132440-8ece2e81-3f91-422d-9cdd-1449f1cf0f86.png)
    + On gist.github.com, looks like this:
        * ![Screen Shot 2022-03-07 at 4 00 51 PM](https://user-images.githubusercontent.com/3385153/157132652-ebf3a100-0d25-4968-8941-0a4eaea52d7f.png)
- Brave Browser (Chromium-based) 1.35.104
    + In repo README.md, looks like this:
        * ![Screen Shot 2022-03-07 at 4 00 51 PM](https://user-images.githubusercontent.com/3385153/157132652-ebf3a100-0d25-4968-8941-0a4eaea52d7f.png)
    + On gist.github.com, looks like this:
        * ![Screen Shot 2022-03-07 at 4 00 51 PM](https://user-images.githubusercontent.com/3385153/157132652-ebf3a100-0d25-4968-8941-0a4eaea52d7f.png)

**Comment by adsteel** [comment]
This is happening for me on a Macbook Pro M2, using Ventura (13.2.1), in a gist, in Firefox, Chrome, and Safari, with even short words.

(Also note the fill colors render differently on different browsers)

## Screenshots

Chrome
<img width="106" alt="image" src="https://user-images.githubusercontent.com/4186462/223744780-c56c5aee-6d09-40ce-9756-a8765854326f.png">

Firefox
<img width="88" alt="image" src="https://user-images.githubusercontent.com/4186462/223744963-34e296a1-a79c-4af4-9c1c-b9a949d2a095.png">

Safari
<img width="94" alt="image" src="https://user-images.githubusercontent.com/4186462/223745088-c7cd0b0e-6392-4e67-aa96-18548996b607.png">

## Live example

```mermaid
graph TB
    classDef orange fill:#ff9966,stroke:#333;

    redis[("<strong>Redis</strong><br><small>(Redis)</small>")]

    redis

    class redis, orange
```
